_tests/integration/docker_test.go

```diff
@@ -63,7 +63,7 @@ func Test_Docker(t *testing.T) {
 			requireErr:   true,
 			requireLogs: []string{
 				"failed to start containers:",
-				"bind: address already in use",
+				"address already in use",
```

Comment:
It seems the error message we expected here has changed, now it looks like:
```
Error response from daemon: failed to set up container networking: driver failed programming external connectivity on endpoint bitrise-workflow-7ff29188-f7e6-4327-83ac-3cc0ae7296de (b2740f9aa549ac5bfb952008f343aedd1cca1979e65d8711efb8580f4e3af269): failed to bind host port for 0.0.0.0:22:172.18.0.2:22/tcp: address already in use
Error: failed to start containers: bitrise-workflow-7ff29188-f7e6-4327-83ac-3cc0ae7296de
```

The end of the error message has changed from `bind: address already in use` to `failed to bind host port for <PORT>/tcp: address already in use`